src/elements/kw-projects-header/kw-projects-header.html

```diff
@@ -112,7 +113,7 @@
                 height: 44px;
                 border-radius: 50%;
                 border: 2px solid #fee33e;
-                z-index: 0;
+                z-index: -1;
```

Comment:
Oh no, I'm so sorry, my bad! I didn't see that Github had collapsed it. Absolutely then, good to go!